SlowCheetah.JDT/JsonDocument.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Sayed Ibrahim Hashimi. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See  License.md file in the project root for full license information.
+
+namespace SlowCheetah.JDT
+{
+    using System;
+    using System.IO;
+    using Newtonsoft.Json;
+    using Newtonsoft.Json.Linq;
+
+    /// <summary>
+    /// Represents a JSON file to be transformed
+    /// </summary>
+    public class JsonDocument : IEquatable<JsonDocument>
```

Comment:
`IEquatable<JsonDocument>` has no use outside of testing. Even then the implementation is very test-specific. Make the JObject publicly accessible and remove `IEquatable`.

It's implementation isn't even corrent: 2 JsonDocuments can be the JObject, but different file paths and they will be equal. That is incorrect.